### DISCUSSION TITLE
字体显示异常

### DISCUSSION THREAD

**Discussion by wulizhong** [Tencent/tgfx#88] [Q&A]
我用安卓demo程序显示字体正常，但是将tgfx按照默认选项编译成静态库加入到自己的项目中，字体显示就不正常了
截图中的文字是 Hello World
![Screenshot_20231230-215851](https://github.com/Tencent/tgfx/assets/2107208/843386cd-6ca9-43e4-83ac-ec1696d440e7)

**Comment by domchen** [comment]
已经有个可以正确运行的demo，找不同这种事需要自己对比。
